frontend/src/metabase/query_builder/components/AlertModals.jsx

```diff
@@ -286,16 +286,9 @@ class UpdateAlertModalContentInner extends Component {
   onAlertChange = modifiedAlert => this.setState({ modifiedAlert });
 
   onUpdateAlert = async () => {
-    const {
-      question,
-      apiUpdateQuestion,
-      updateAlert,
-      updateUrl,
-      onAlertUpdated,
-    } = this.props;
+    const { question, updateAlert, updateUrl, onAlertUpdated } = this.props;
     const { modifiedAlert } = this.state;
 
-    await apiUpdateQuestion();
```

Comment:
This is the core behavioral change in this PR: when we update an alert, we don't also update the question.

It might once have made sense to automatically update the question when the alert is updated, but no longer, I think, because it is now impossible to update an alert if the corresponding question has not yet been saved.